buildchain/buildchain/salt_tree.py

```diff
@@ -59,6 +59,46 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
         yield from file_tree.execution_plan
 
 
+class ConstantContainerRegistry(targets.AtomicTarget):
+    """Generate a nginx configuration to serve constants container registry."""
+    def __init__(
+        self,
+        destination: Path,
+        **kwargs: Any
+    ):
+        """Configure the static-container-registry script.
```

Comment:
Not really, you just specify the output path (which is not used by the script).

Reply:
True just copy paste, I will change this one